.github/ISSUE_TEMPLATE/02_bug.md

```diff
@@ -18,13 +21,16 @@ How can you reproduce the bug?
 
 
 ### How are you running the application?
-Please include at least:
+Please include as many of the following as possible:
 - **Zulip-terminal version:**
   eg. a specific version (0.7.0), or if running from `main` also ideally the git ref
-- **Zulip server version:**
-  eg. Zulip Cloud, or the version you are running self-hosted
+- **Zulip server version(s):**
+  eg. Zulip Cloud, the version you are running self-hosted, or the Zulip Community server (chat.zulip.org)
 - **Operating system (and version):**
-  eg. Debian Linux, Ubuntu Linux, macOS, WSL in Windows
+  eg. Debian Linux, Ubuntu Linux, macOS, WSL in Windows, Docker
+- **Python version (and implementation):**
+  eg. 3.8, 3.9, 3.10, ... (implementation is likely to be eg. CPython, or PyPy)
 
-If possible, please provide details from the `About` menu:
+If possible, please provide details from the `About` menu: (Meta + ?)
```

Comment:
We might want to mention or clarify that the part in the brackets is the hotkey to access it.
It may not be obvious, as 'meta' and '?' are not just keys.

Reply:
I dropped the `<kbd>` tags from the text as discussed previously, since they are only visible when previewed/submitted, so are a bit messy when filling out the original box.

Just added a `hotkey: ` prefix before the keys.